src/styles.css

```diff
@@ -176,6 +176,12 @@ table.scroll tbody {
   }
 }
 
+@media screen and (min-width: 769px) {
```

Comment:
May be better to keep it components specific. South/ North/ Notify/ View Config Item components etc.